template/runtime/src/lib.rs

```diff
@@ -448,6 +449,32 @@ impl_runtime_apis! {
 		fn transaction_status(hash: H256) -> Option<ethereum::TransactionStatus> {
 			ethereum::Module::<Runtime>::transaction_status(hash)
 		}
+
+		fn gas_price() -> U256 {
+			FixedGasPrice::min_gas_price()
+		}
+
+		fn block_number() -> U256 {
+			let number: u128 = <system::Module<Runtime>>::block_number().unique_saturated_into();
+			U256::from(number)
+		}
+
+		fn evm_balance(address: H160) -> U256 {
+			if !evm::Module::<Runtime>::account_exists(&address) {
```

Comment:
No need for `account_exists`, just remove this if branch -- empty accounts are handled in evm module.